### ISSUE TITLE
Navigation

### ISSUE THREAD

**Issue by nadjaweisskopf** [leihs/leihs#888]
Navigation: 

Describe the goal of the navigation - then the detailed description of navigation elements

I would suggest to place following pages in the main navigation on the top right: 

- cart
- favorites
- orders

All other pages are available through the dropdop menu.

**Comment by nadjaweisskopf** [comment]
Nur design, nichts für Backend

**Comment by eins78** [comment]
@urs-van-binsbergen will integrate the design spec into the app and take care of some layout inconsistencies so it works everywhere

- [ ] design menu
- [ ] integrate with Navbar component
- [ ] integrate in app

**Comment by urs-van-binsbergen** [comment]
Branch `uvb/menu`

Discussion points
- Menu is collapsed on `:routing/change-view` event, which is also fired on page (re-)load. Can and should we change this, i.e. only fire on a real "navigate"? Otherwise I will remove the state from local storage.
- ErrorBoundary
  - It is now integrated in `PageLayout`
  - It seems to work fine, I tested it with a button setting a "crash atom" :) 
  - Note that the story was broken (shame on me, I moved the crash outside of the boundary when making it button-triggered)
